src/handlers/rfqm_handlers.ts

```diff
@@ -183,6 +284,25 @@ export class RfqmHandlers {
     }
 }
 
+const parseSubmitSignedQuoteParams = (req: express.Request): SubmitRfqmSignedQuoteParams => {
+    const type = req.query.type as RfqmTypes;
+
+    if (type === RfqmTypes.MetaTransaction) {
+        const metaTransaction = new MetaTransaction(
+            stringsToMetaTransactionFields((req.query.metaTransactionas as unknown) as StringMetaTransactionFields),
+        );
+        const signature = stringsToSignature((req.query.signature as unknown) as StringSignatureFields);
+
+        return {
+            type,
+            metaTransaction,
+            signature,
+        };
+    } else {
+        throw new Error('Unsupported submission type for /submit');
```

Comment:
How do these errors bubble up to the customer?

Reply:
Can we use 4xx status code?

Reply:
Could we use a `BadRequestError` for this? If not, a `NotImplementedError` might be a good option